lib/cosmos/gui/qt_tool.rb

```diff
@@ -32,21 +32,57 @@ class QtTool < Qt::MainWindow
     def initialize(options)
       # Call QT::MainWindow constructor
       super() # MUST BE FIRST - All code before super is executed twice in RubyQt Based classes
+      @options = options
+      @about_string = nil
 
       # Add Path for plugins
       Qt::Application.instance.addLibraryPath(Qt::PLUGIN_PATH) if Kernel.is_windows?
-
       # Prevent killing the parent process from killing this GUI application
       Process.setpgrp unless Kernel.is_windows?
 
-      self.class.redirect_io if options.redirect_io
+      self.class.redirect_io if @options.redirect_io
+      self.window_title = @options.title
+      Cosmos.load_cosmos_icon
 
-      # Configure instance variables
-      @options = options
-      @about_string = nil
+      # Read the application wide stylesheet if it exists
+      app_style = File.join(Cosmos::USERPATH, 'config', 'tools', 'application.css')
+      @stylesheet = ''
+      @stylesheet = File.read(app_style) if File.exist? app_style
```

Comment:
After starting to create a bunch of individual stylesheets I quickly realized how nice it would be to have an global style. I read it into a variable and then read application specific sheets later. They override the global style because they come after. Seems to work pretty good.